src/nile/cli.py

```diff
@@ -89,26 +111,42 @@ def run(path, network):
 @network_option
 @click.option("--alias")
 @click.option("--abi")
-def deploy(artifact, arguments, network, alias, abi=None):
+@track_option
+@debug_option
+def deploy(artifact, arguments, network, alias, track, debug, abi=None):
     """Deploy StarkNet smart contract."""
-    deploy_command(artifact, arguments, network, alias, abi=abi)
+    deploy_command(
+        contract_name=artifact,
+        arguments=arguments,
+        network=network,
+        alias=alias,
+        abi=abi,
+        track=track,
+        debug=debug,
+    )
 
 
 @cli.command()
 @click.argument("artifact", nargs=1)
 @network_option
 @click.option("--alias")
-def declare(artifact, network, alias):
+@track_option
+@debug_option
+def declare(artifact, network, alias, track, debug):
     """Declare StarkNet smart contract."""
-    declare_command(artifact, network, alias)
+    declare_command(
+        contract_name=artifact, network=network, alias=alias, track=track, debug=debug
+    )
 
 
 @cli.command()
 @click.argument("signer", nargs=1)
 @network_option
-def setup(signer, network):
+@track_option
+@debug_option
+def setup(signer, network, track=False, debug=False):
     """Set up an Account contract."""
-    Account(signer, network)
+    Account(signer, network=network, track=track, debug=debug)
```

Comment:
A small suggestion to avoid to much verbosity when is possible:
```suggestion
    Account(signer, network, track=track, debug=debug)
```